src/screens/index.js

```diff
@@ -1,31 +1,19 @@
 import AddParticipant from './AddParticipant';
+import AnonymousChannelListScreen from './AnonymousChannelListScreen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `AnonymousChannelListScreen` is new, but there's no removal of `ChannelListScreenV2` in the old hunk. This could potentially lead to unused code if `ChannelListScreenV2` is not used elsewhere.
```diff
- import AnonymousChannelListScreen from './AnonymousChannelListScreen';
+ import ChannelListScreenV2 from './ChannelListScreenV2';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->